### ISSUE TITLE
Change color scheme doesn't work in ST4

### ISSUE THREAD

**Issue by igor-kupczynski** [SublimeText-Markdown/MarkdownEditing#595]
When selecting `MarkdownEditing: Change color scheme ...` in ST4 on macOS it opens the file, but doesn't show the theme preview window:

```python
Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/Lib/python33/sublime_plugin.py", line 1494, in run_
    return self.run(edit)
  File "/Users/igor/Library/Application Support/Sublime Text 3/Installed Packages/MarkdownEditing.sublime-package/bootstrap.py", line 158, in run
  File "/Users/igor/Library/Application Support/Sublime Text 3/Installed Packages/MarkdownEditing.sublime-package/bootstrap.py", line 105, in choose_color_theme
  File "/Users/igor/Library/Application Support/Sublime Text 3/Installed Packages/MarkdownEditing.sublime-package/bootstrap.py", line 105, in <listcomp>
AttributeError: 'NoneType' object has no attribute 'group'
```

**Comment by tukusejssirs** [comment]
It doesn’t work on Linux either. I have just installed the plugin, it complains about `error: Error loading syntax file "Packages/Markdown/Markdown.sublime-syntax": Unable to read Packages/Markdown/Markdown.sublime-syntax`. I tried restarting ST4, but it does not fix the issue.

My issue might be a bit different though. After I installed the plugin via Package Control, it _didn’t_ create any folder under `Packages/`, i.e. neither `Markdown/` nor `MarkdownEditing/`, therefore I seems like not all files are downloaded and placed to their respective locations.

BTW, I have installed `v2.2.9`.
